eslint.config.js

```diff
@@ -0,0 +1,17 @@
+import importPlugin from "eslint-plugin-import";
+import prettier from "eslint-plugin-prettier/recommended";
+
+export default [
+  importPlugin.flatConfigs.recommended,
+  {
+    files: ["src/**/*.{js,mjs}"],
```

Comment:
Should we not lint all files in the repository? I think we can remove this and fall back to the [default behavior](https://eslint.org/docs/latest/use/configure/configuration-files#specifying-files-and-ignores):

> By default, ESLint lints files that match the patterns `**/*.js`, `**/*.cjs`, and `**/*.mjs`.

This is also what we do in `file-selector`.

```suggestion
```

Reply:
I've removed this section.